docs/nix.md

```diff
@@ -0,0 +1,233 @@
+# Using Nix with Cardano SL
+
+We use Nix for building and deploying Cardano SL. It provides fully
+deterministic and cached builds of our software and all dependencies,
+and permits fully declarative configuration of the infrastructure
+running the Cardano SL network.
+
+## Installing
+
+The [Nix](https://nixos.org/nix/) package manager can be installed on
+any Linux distribution (with SELinux disabled) or on macOS.
+
+Multi user mode is the best and most commonly tested configuration. To
+install Nix in multi user mode:
+
+    sh <(curl https://nixos.org/nix/install) --daemon
+
+Full instructions are in the
+[Nix Manual](https://nixos.org/nix/manual/#ch-installing-binary).
+
+## Binary cache
+
+Configuring the IOHK binary cache is essential for developing with
+Cardano SL. If you do not have the binary cache set up correctly, Nix
+will download source tarballs and proceed to build an entire system
+from scratch.
+
+Add the following lines to `/etc/nix/nix.conf`. If the file does not
+exist, then create it.
+
+    substituters         = https://hydra.iohk.io https://cache.nixos.org
+    trusted-substituters =
+    trusted-public-keys  = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+
+The `nix-daemon` must be restarted after editing `/etc/nix/nix.conf`
+for changes to take effect. Run `systemctl restart nix-daemon` on Linux
+or `sudo launchctl stop org.nixos.nix-daemon; sudo launchctl start
+org.nixos.nix-daemon` on macOS.
+
+We do not recommend using `~/.config/nix/nix.conf` unless you are a
+power user. It's simpler to have only one global config file.
+
+If using NixOS, see [`iohk-binary-cache.nix`](../nix/iohk-binary-cache.nix).
```

Comment:
This link does not resolve, unless I'm missing something?

Reply:
Fixed, thanks.